### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by san00**

### CODE REVIEW

File: src/sass/style.scss [hunk 1]
```diff
@@ -40,7 +40,24 @@ html {
...
 }
-
+.banner-text{
+    position: absolute;
+    bottom: 25rem;
+    left: 20rem;
+    background-color: white;
+    width: 25rem;
+    height: 10rem;
+    border: 0.5rem solid lightgray;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 2]
```diff
@@ -40,7 +40,24 @@ html {
     object-fit: cover;
     width: 100vw;
 }
-
+.banner-text{
+    position: absolute;
+    bottom: 25rem;
+    left: 20rem;
+    background-color: white;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 3]
```diff
@@ -40,7 +40,24 @@ html {
...
+    height: 10rem;
+    border: 0.5rem solid lightgray;
+}
+.text{
+    text-align: center;
+}
+.banner-text-main, {
+    font-size: 1.5rem;
+    padding-top:3rem;
+    font-weight: bold;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Remember not to use bold, use 700 (look for faux web typography and Zach Letherman).

File: src/sass/style.scss [hunk 4]
```diff
@@ -68,6 +68,15 @@ html {
 
 .work-navbar--links {
     text-decoration: none;
+    color: grey;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 5]
```diff
@@ -15,81 +15,92 @@ html {
 }
 
 .main-navbar {
+    background-color: aquamarine;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 6]
```diff
@@ -15,81 +15,92 @@ html {
...
     display: flex;
     margin-bottom: 3rem;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Review every margin and padding in the document and use spacing variables

File: src/sass/style.scss [hunk 7]
```diff
@@ -15,81 +15,92 @@ html {
...
-.banner-text{
-    position: absolute;
-    bottom: 25rem;
-    left: 20rem;
+
+.banner-text {
     background-color: white;
-    width: 25rem;
-    height: 10rem;
     border: 0.5rem solid lightgray;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 8]
```diff
@@ -15,81 +15,92 @@ html {
...
+.banner-text-all {
+    font-size: 1.2rem;
     text-align: center;
 }
-.banner-text-main, {
-    font-size: 1.5rem;
-    padding-top:3rem;
+
+.banner-text-header {
     font-weight: bold;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Don't use bold use 700 (search for Faux as previously)

File: src/sass/style.scss [hunk 9]
```diff
@@ -15,81 +15,92 @@ html {
...
-.work-navbar--links {
-    text-decoration: none;
+.portfolio-navbar--links {
     color: grey;
+    text-decoration: none;
 }
-.work-navbar--links:hover{
+
+.portfolio-navbar--links:hover {
     background-color: black;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 10]
```diff
@@ -15,37 +26,37 @@ html {
...
-    margin-bottom: 2rem;
+    margin-bottom: $spacing-l;
     object-fit: cover;
     width: 100vw;
 }
 
-.banner-text {
+.banner__text {
     background-color: white;
     border: 0.5rem solid lightgray;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 11]
```diff
@@ -55,65 +66,63 @@ html {
...
-    margin-left: 5rem;
-    margin-right: 5rem;
+    margin-bottom: $spacing-l;
+    margin-left: $spacing-xl;
+    margin-right: $spacing-xl;
 }
 
-.portfolio-navbar--links {
+.portfolio__navbar-links{
     color: grey;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 12]
```diff
@@ -55,65 +66,63 @@ html {
...
     text-decoration: none;
 }
 
-.portfolio-navbar--links:hover {
+.portfolio__navbar-links:hover {
     background-color: black;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 13]
```diff
@@ -55,65 +66,63 @@ html {
...
     color: white;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss [hunk 14]
```diff
@@ -55,65 +66,63 @@ html {
...
-    height: 35vh ;
+.portfolio__images-gallery {
+    height: 35vh;
     object-fit: cover;
-    width: 35vw ;
+    width: 35vw;
 }
 
 .footer {
     background-color: black;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use color as variable

File: src/sass/style.scss (hunk 3 above)
Comment (san00) [reply src/sass/style.scss]: Will do. Thanks.

Comment (san00) [reply src/sass/style.scss]: I'll look this up. Thanks

**Pull request merged**